Sources/MongoSwift/BSON/Document+Codable.swift

```diff
@@ -1,219 +1,56 @@
 import Foundation
 
+/// An extension of `Document` to implement the `Codable` protocol.
 extension Document: Codable {
 	public func encode(to encoder: Encoder) throws {
+		// if we're using a `BsonEncoder`, we can just short-circuit
```

Comment:
FYI: This entire file appears to be indented with tab characters instead of four spaces.